### ISSUE TITLE
the vis-functions are not called

### ISSUE THREAD

**Issue by oweitman** [ioBroker/ioBroker.vis#454]
**Describe the bug**  
in some cases the binding expression is interpreted as eval. then, the resulting javascript want to call the vis function which is not available.

**To Reproduce**  
1. create a datapoint with name 0_userdata.0.test1
2. set value to ""181.3999999"" or "181.3999999"
3. in vis create a html-widget
4. as html insert the following expression
{v1:0_userdata.0.test1;parseFloat(v1);round(1)}
5. in runtime, the result is 0

**Expected behavior**  
the result should be 181.4

**Screenshots & Logfiles**  
browser log (callstack removed)
```
instrument.ts:129 Error in eval[value]     : {v1:0_userdata.0.test1;parseFloat(v1);round(1)}
instrument.ts:129 Error in eval[script]: var v1 = "181.3999999";return round(1);
instrument.ts:129 Error in eval[error] : ReferenceError: round is not defined
```

**Versions:**  
vis 1.3.4

https://forum.iobroker.net/topic/12319/rechnen-in-widgets/50?_=1644327092076

**Comment by Apollon77** [comment]
Yes, the function is called Math.round ... there is no "round" in Javascript

**Comment by Apollon77** [comment]
An in fact yes bindings are converted into mini javsscripts and eval'ed ...

**Comment by oweitman** [comment]
yes, there is no round in javascript, but vis owns a binding-function named round
![image](https://user-images.githubusercontent.com/514484/153092326-3a6509fe-7f98-49c1-8175-f72100c215b6.png)
and these functions should be usable as described in the documentation
i guess the same is with the other functions
https://github.com/ioBroker/ioBroker.vis#bindings-of-objects -> Following operations are supported
please reopen

**Comment by Apollon77** [comment]
Oohh, ok learned something new :-)) @GermanBluefox ?

**Comment by oweitman** [comment]
Still relevant
